modules/ossm-extensions-wasmplugin-deploy.adoc

```diff
@@ -0,0 +1,41 @@
+////
+This module included in the following assemblies:
+*service_mesh_/v2x/ossm-extensions.adoc
+////
+:_content-type: PROCEDURE
+[id="ossm-wasmplugin-deploy_{context}"]
+= Deploying `WasmPlugin` resources
+
+You can enable {SMProductName} extensions using the `ServiceMeshExtension` resource. In this example, `istio-system` is the name of the control plane project.  The following example creates an `openid-connect filter` that performs an OpenID Connect flow to authenticate the user.
```

Comment:
Replace `ServiceMeshExtension` with `WasmPlugin`

Reply:
D'oh, copy/paste error!  Thanks for catching that.